README.md

```diff
@@ -1 +1,62 @@
 # Manifest Schema
+
+This project contains publicly available json schemas that define slacks `manifest.json` file. JSON schemas can be used for a number of tasks but the purpose of this specific schema is to provide developers validation and prediction capabilities while editing `manifest.json` files in IDEs.
+
+**Guide Outline**:
+
+- [Manifest Schema](#manifest-schema)
```

Comment:
I would add a Requirements section. What runtimes/languages/tooling do I need to run this project on my laptop? Looking at the scripts, seems like I need python/pip, as well as npx, so, which version of python should I have? Does this work with both Python 2.x as well as 3.x? Should I set up a virtual env to work in this project? Which version of node/npm/npx should I have?

Reply:
I've placed this information in the `.github/maintainers_guide.md`,

I put this information there since it should only be require by the maintainers of this project. Users of the schemas should not need to use this Python or Node. Python and Node are used to Lint, Format and Test the project.